peachjam/models/core_document_model.py

```diff
@@ -324,13 +324,22 @@ class AttachmentAbstractModel(models.Model):
     filename = models.CharField(max_length=1024, null=False, blank=False)
     mimetype = models.CharField(max_length=1024, null=False, blank=False)
     size = models.BigIntegerField(default=0)
+    file = models.FileField()
 
     def __str__(self):
         return f"{self.filename}"
 
     class Meta:
         abstract = True
 
+    def save(self, *args, **kwargs):
```

Comment:
The similar code in `SourceFile` can fall away.

Reply:
removed. ✔️